src/api/kotlin/xyz/wagyourtail/unimined/api/mapping/MappingsConfig.kt

```diff
@@ -243,6 +259,38 @@ abstract class MappingsConfig(val project: Project, val minecraft: MinecraftConf
         yarn(build.toInt(), key, action)
     }
 
+    @JvmOverloads
+    abstract fun feather(build: Int, key: String = "yarn", action: MappingDepConfig.() -> Unit = {})
```

Comment:
I'm not familiar with Unimined internals, but what is the purpose of this `"yarn"` string here?

Reply:
if I remember correctly it's the mappings kind/type

Reply:
I see - that's used to tell Unimined to unpack and parse the artifacts the way it does Yarn, for instance, I imagine

Reply:
> I see - that's used to tell Unimined to unpack and parse the artifacts the way it does Yarn, for instance, I imagine

Correct.